rsmt2d_test.go

```diff
@@ -0,0 +1,61 @@
+package rsmt2d_test
```

Comment:
Nice! I think we should move most of the tests into this package. Can happen in a followup PR.

We should open an issue about testing (test-coverage and this) if it doesn't exist yet.

Reply:
Opened #58.